Cargo.toml

```diff
@@ -43,8 +43,10 @@ instrument = ["rftrace", "rftrace-frontend"]
 
 [dependencies]
 align-address = "0.3.0"
+bimap = "0.6.3"
```

Comment:
This dependency might have to be replaced: We need a *vector* to address a container that contains multiple file descriptors.